skyportal/models.py

```diff
@@ -96,23 +98,37 @@ class Group(Base):
 
 class Stream(Base):
     name = sa.Column(sa.String, unique=True, nullable=False)
-    url = sa.Column(sa.String, unique=True, nullable=False)
-    username = sa.Column(sa.String)
-    password = sa.Column(sa.String)
+    altdata = sa.Column(JSONB, nullable=True,
```

Comment:
`altdata` doesn't seem like the right place for this. Should we also be keeping track of what instrument a stream is connected to, so we can prevent users from implementing filters that run on streams from multiple instruments / collections, if that is not supported on K?

Reply:
I originally had two columns instead, `collection` and `selector`, but decided to move them to `altdata` (thanks to Josh for the suggestion!) to keep it general enough to be included in general SP. 

The users won't be able to do that: a filter is defined in a group for a single stream, and only the System admin can create new streams.